consumer/pdata/generated_metrics.go

```diff
@@ -607,6 +607,8 @@ func (ms Metric) SetUnit(v string) {
 	(*ms.orig).Unit = v
 }
 
+
+
```

Comment:
I re-run `make pgendata` and this is the result. Probably a previous change forgot to run it.

Reply:
Fix in https://github.com/open-telemetry/opentelemetry-collector/pull/1740, if you run `make genproto` again you will get this fixed, anyway I created the PR to fix this in makefile

Reply:
It appears to be because of this line https://github.com/open-telemetry/opentelemetry-collector/blob/315e96ecb6d7249d9cf74496a0b68fe7f7bd64da/cmd/pdatagen/internal/base_structs.go#L437
Should that be deleted instead?

Reply:
it is not about that line, that line is ok for 99% of the cases, except the "hacky" oneof field which does not have a getter/setter because the code was manually written

Reply:
Got it.